packages/sdk-react/src/components/misc/FilterContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactNode } from 'react';
+
+import { classNames } from '@/utils/css-utils';
+import { Stack } from '@mui/material';
+
+interface Props {
```

Comment:
We have a convention to name them as `ComponentNameProps`, so in this case `FilterContainerProps`